pkg/plugin/plugin.go

```
@@ -151,6 +151,18 @@ func splitIDs(s string) []string {
 	return ids
 }
 
+// defaultTimeColumn returns the column the query editor preselects as the time
+// field, given the table's columns keyed by name with their CQL type.
+func defaultTimeColumn(columns map[string]string) string {
```

golangci-lint (2.12.2)

[error] func defaultTimeColumn is unused

(unused)